explainers/audio-explainer.md

```diff
@@ -1,45 +1,76 @@
 
 Audiobooks have grown massively in popularity in the last few years despite being a segment of the industry for far longer. The main reason for this shift is due to several technologies:
-* Mobile devices (phones and tablets)
-* Improved mobile networks (3G, 4G LTE)
+* Pervasiveness of Mobile devices (phones and tablets)
 * Improved mobile storage capacity
+* Improved mobile networks (3G, 4G, LTE)
 * Widespread use of wifi
 
-When previously audiobook users would need several CDs to listen to their audiobook, they can now stream, download, and listen to their chosen content anywhere and time they choose. Audiobooks, like ebooks, are now in their pockets on demand. 
+When previously users would need several CDs to listen to one audiobook, they can now stream or download and listen to their chosen content anywhere, any time. Audiobooks, like ebooks, are now in their pockets on demand. 
 
-The main difference between ebooks and audiobooks currently is their level of specification. EPUB has existed in one form or another for almost 20 years, but audiobooks still do not have a common specification. A mature ebooks industry has co-opted audiobooks to make them work within the same contexts, but the lack of a specification has made this process arduous for user agents and by extension, users. 
+The main difference between ebooks and audiobooks is currently their level of specification. The EPUB format has existed as an open standard in one form or another for almost 20 years, but audiobooks still do not benefit from a standard format. A mature publishing industry has co-opted audiobooks to make them work within the same contexts, but the lack of standardization has made this process arduous for audiobook studios, distributors, retail platforms, user agents and by extension, users. 
+
+## Key use cases and current issues
+
+### Business to business
+
+Publishers usually team up with specialized studios to produce audiobook content. Once produced, each audiobook is sent to different distributors and retail platforms. Distributor are themselves hubs in the supply chain, which redirect the audiobooks to multiple retail (or public lending) platforms.
+
+Today, audio content is split into a number of mp3 files simply ordered by their file name. Each audiobook is then usually packaged in a zip file. The quality and size of these files is constrained by the target distributors. Because of these constraints, an audio track may not correspond to any logical section of the book. Metadata attached to the audio content is sparse, limited to what the mp3 format can handle; even then, numerous metadata interoperability issues down the line force studios to apply tight constraints, different for each distributor, like using only ascii capital letters on authors names. 
+
+This results in duplicated effort, increases the chances of errors, and makes the supply chain less efficient.
+
+### Audiobooks for print disabled people
```

Comment:
Similar to above, we can outline DAISY talking books further in the alternate formats section.